platform/macos/detect.py

```diff
@@ -97,6 +89,7 @@ def configure(env: "SConsEnvironment"):
         env.Append(LINKFLAGS=["-arch", "x86_64", "-mmacosx-version-min=10.13"])
 
     env.Append(CCFLAGS=["-ffp-contract=off"])
+    env.Append(CCFLAGS=["-fobjc-arc"])
```

Comment:
```suggestion
    env.Append(CCFLAGS=["-ffp-contract=off", "-fobjc-arc"])
```

Reply:
They're not really related so I think it makes sense to keep them separate (we might remove one independently from the other).